webapp/components/_new/features/SearchResults/SearchResults.vue

```diff
@@ -59,6 +59,14 @@
               </base-card>
             </ds-grid-item>
           </template>
+          <!-- groups -->
+          <template v-if="activeTab === 'Group'">
+            <ds-grid-item v-for="group in activeResources" :key="group.id" :row-span="2">
+              <base-card :wideContent="true" class="group-teaser-card-wrapper">
+                <group-teaser :group="{ ...group, name: group.groupName }" />
```

Comment:
Location is not even used on the group profile, so the teaser does not need it ether. The `groupName` is needed because the union of groups and users both have name

Reply:
Sorry, but the `group.location` is used on the group profile and on the group teaser dear.

I tried it out and the location appears in the group teaser in the search results if we request it from the db.

Reply:
OK. But this is not working as expected. It looks like the group query does not always return the location field, as it is not queried at all. If have no idea why sometimes a location is rendered